src/app/my-account/my-account.component.ts

```diff
@@ -51,7 +54,15 @@ export class MyAccountComponent implements OnInit {
       }
     });
 
-    setTimeout(this.checkForPaymentCardsIfNotLoaded, 5_000);
+    if (isPlatformBrowser(this.platformId)) {
```

Comment:
not a blocker, but might be worth moving this inside the `else` block above, so it doesn't run unless there is a logged in person and after we know who they are.